### PR TITLE
Fixed the code to remove unwanted escape characters in JSON

### PR DESCRIPTION
**Pull request by pravilla**
When the input is a JSON document, the transformer function was treating double quotes as part of the string and was adding an escape sequence (\) to the double quote.

Modified the transformer function to remove the .stringify() function.

### CODE REVIEW

File: transformer.js [hunk 1]
```diff
@@ -21,7 +21,8 @@ exports.addNewlineTransformer = addNewlineTransformer;
 /** Convert JSON data to its String representation */
 function jsonToStringTransformer(data, callback) {
     // emitting a new buffer as text with newline
-    callback(null, new Buffer(JSON.stringify(data) + "\n", targetEncoding));
+    //callback(null, new Buffer(JSON.stringify(data) + "\n", targetEncoding));
```
Comment (benoittgt) [inline transformer.js]: Do you think we need to keep the previous line commented ?

File: test/testInputs.test.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+var assert = require('assert');
```
Comment (benoittgt) [inline test/testInputs.test.js]: Thanks for the tests

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
 	"name": "lambda-stream-to-firehose",
 	"description": "An AWS Lambda function that forwards data from a Kinesis or DynamoDB Update Stream to a Kinesis Firehose Delivery Stream",
-	"version": "1.4.3",
+	"version": "1.4.3-offerup",
```
Comment (benoittgt) [inline package.json]: What is this ?

**Comment by bnayalivne** [comment]
the build for version 1.4.4 is broken, it does not contain this bugfix. took me time to understand that the "dist" version is not the same as the code.

**Comment by benoittgt** [comment]
@bnayalivne you should use the build script with the master of the lambda https://github.com/awslabs/lambda-streams-to-firehose#debugging--creating-new-builds

**Pull request merged**